docs/content/en/tasks/monitor/prometheus-grafana/_index.md

```diff
@@ -144,12 +144,47 @@ test-dc1-stargate-stargate-servicemonitor            5m47s
 ### Install the Grafana dashboards
 
 Grafana will pick up dashboards passed as configmaps that have the label `grafana_dashboard: "1"`.  
-Create the overview, condensed and stargate dashboards (download the manifest [here](grafana-dashboards.yaml)) configmaps:
+**When using MCAC**, create the overview, condensed and stargate dashboards (download the manifest [here](grafana-dashboards.yaml)) configmaps:
 
 ```bash
 kubectl apply -f grafana-dashboards.yaml -n k8ssandra-operator
 ``` 
 
+**When using the new metrics endpoint** (and MCAC is disabled), create the overview, condensed and stargate dashboards (download the manifest [here](grafana-dashboards-new.yaml)) configmaps:
+
+```bash
+kubectl apply -f grafana-dashboards-new.yaml -n k8ssandra-operator
+```
+
+The new metrics endpoint doesn't extract os level metrics (cpu, memory, disk, etc) so the overview dashboard will be missing those metrics by default. The Prometheus node_exporter can be deployed to expose those metrics for scraping, and the corresponding charts will have to be adjusted accordingly in the dashboard.  
+An alternative is to enable Vector instead and include the following components in the Vector configuration:
```

Comment:
This assumes user must understand Go struct -> YAML to use this information. Could we just provide yaml here?

Reply:
This is open source repo @Miles-Garnsey ;)

Reply:
Hah, now I'm the one getting confused as to which repo we're in. In that case, that being the case, I kind of stand by keeping the API references but agree that maybe we should add a yaml example since we don't get as much docs support over here in OSS land.

Reply:
> This assumes user must understand Go struct -> YAML to use this information. Could we just provide yaml here?

Good point!
I'll provide the yaml representation as well.

Reply:
Done! (finally...)